### ISSUE TITLE
Calibrated sensor information

### ISSUE THREAD

**Issue by astecken** [aodn/aatams#338]
Users are interested in the following enhancement:
Detections downloaded via the web app don’t take into account the calibration information you put in for each tag so it is unfortunately impossible at this stage to download calibrated sensor values. 
What needs to be done currently: 
What I could do for you Kate though is provide you with a csv that has the intercept and slope for each of your tags so that you can then apply the calibration yourself.

**Comment by fjaine** [comment]
YEs, I agree that this is a fairly high priority issue - I have received an email related to this a few days ago from someone at NSW DPI:

**Comment by fjaine** [comment]
@xhoenner this is still a high priority issue - I have recently received an email related to this from someone at NSW DPI: 

There appears to be an issue with the calibration of sensor tags in the database.  When I extracted the Bull shark detections from receivers in Sydney Harbour, the sensor tags are uncalibrated.  Is there anyway of correcting this?
